ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -345,11 +345,13 @@ local_users:
     role: < role >
     sha512_password: "< sha_512_password >"
     no_password: < true | do not configure a password for given username. sha512_password MUST not be defined for this user. >
+    sshkey: "< ssh-key_string >"
   < user_2 >:
     privilege: < 1-15 >
     role: < role >
     sha512_password: "< sha_512_password >"
     no_password: < true | do not configure a password for given username. sha512_password MUST not be defined for this user. >
+    sshkey: "< ssh-key_string >"
```

Comment:
to be consistent with the rest of the data model, could you update `sshkey` to `ssh_key`. 
We have tried to apply conversion from `-` in EOS to `_` in YAML as much as possible.

Reply:
Both doc and YAML have been updated to reflect the appropriate naming convention. Thanks for the heads up, will take note of this in future.

Reply:
Great !